README.md

```diff
@@ -4,132 +4,150 @@
+||||
+|| Make Juju your own! | Write your own charm: <br> https://juju.is/docs/sdk <br> _Juju is written in Go, but our SDK supports easy charm development in Python!_ |
+
+## Give it a try!
+
+### Set up
+
+You will need a cloud and Juju. The quickest way is to use a Multipass VM launched with the `charm-dev` blueprint. 
+
+Install Multipass: [Linux](https://multipass.run/docs/installing-on-linux) | [macOS](https://multipass.run/docs/installing-on-macos) | [Windows](https://multipass.run/docs/installing-on-windows). On Linux:
+
+```
+sudo snap install multipass
+```
+
+Use Multipass to launch an Ubuntu VM with the `charm-dev` blueprint: 
+
+```
+multipass launch --cpus 4 --memory 8G --disk 30G --name tutorial-vm charm-dev 
+```
+
+Open a shell into the VM:
+
+```
+multipass shell tutorial-vm
+```
+
+Verify that you have Juju and two localhost clouds:
+
+```
+juju clouds
+```
+
+Bootstrap a Juju controller into the MicroK8s cloud:
+
+```
+juju bootstrap microk8s tutorial-controller
+```
+
+Add a workspace, or 'model':
+
+```
+juju add-model tutorial-model
+```
+
+### Deploy, configure, and integrate a few things
+
+Deploy Mattermost:
+
+```
+juju deploy mattermost-k8s
+```
+> See more: [Charmhub | `mattermost-k8`](https://charmhub.io/mattermost-k8s) 
+
+Deploy PostgreSQL:
+
+```
+juju deploy postgresql-k8s --channel 14/stable --trust
+```
+
+> See more: [Charmhub | `postgresql-k8s`](https://charmhub.io/postgresql-k8s)
+
+Enable security in your PostgreSQL deployment:
+
+```
+juju deploy tls-certificates-operator
+juju config tls-certificates-operator generate-self-signed-certificates="true" ca-common-name="Test CA"
+juju integrate postgresql-k8s tls-certificates-operator
+```
+
+Integrate Mattermost with PostgreSQL:
+
+```
+juju integrate mattermost-k8s postgresql-k8s:db
+```
+
+Watch your deployment come to life:
+
+```
+juju status --watch 1s
+```
+
+(Press Ctrl-C to quit. Drop the `--watch 1s` flag to get the status statically. Use the `--relations` flag to view more information about your integrations.)
+
+### Test your deployment
+
+When everything is in `active` or `idle` status, note the IP address and port of Mattermost and pass them to `curl`:
+
+```
+curl <IP address>:<port>/api/v4/system/ping
+```
+
+You should see the output below:
+
+```
+{"AndroidLatestVersion":"","AndroidMinVersion":"","IosLatestVersion":"","IosMinVersion":"","status":"OK"}
+```
+### Congratulations!
+
+You now have a Kubernetes deployment consisting of a Mattermost backed by PosgreSQL with TLS-encrypted traffic!
+
+### Clean up
+
+Delete your Multipass VM:
+
```

Comment:
You may want to include `juju destroy-controller tutorial-controller --destroy-all-models` here for those who aren't working in a vm

Reply:
Well, if they decide not to use the VM, they're on their own anyway, because we don't show them how to set up the cloud, install Juju, etc., anyway. WDYT?

Reply:
Yeah fair enough. If you're happy I'm happy